virttest/qemu_vm.py

```diff
@@ -4512,7 +4528,10 @@ def resume(self, timeout=None):
         """
         self.monitor.cmd("cont")
         if timeout:
-            self.wait_for_status('running', timeout, 0.1)
+            if not self.wait_for_status('running', timeout, 0.1):
+                raise virt_vm.VMStatusError("Failed to enter running status, "
+                                            "the actual status is %s" %
+                                            self.monitor.get_status())
```

Comment:
Nice, this is actually a bugfix and not related to the current patch. Would you please extract it as a separate commit?

Reply:
Yes, that is, make it as an separate commit is better.